### PR TITLE
feat: add avatar component

### PR DESCRIPTION
**Pull request by rahulmistry751**
Added avatar component
- Rounded avatar
- Square avatar

image
image

### CODE REVIEW

File: components/css/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+.avatar{
+    border-radius:50%;
+    width:48px;
+    height:48px;
```
Comment (znjs) [inline components/css/avatar/avatar.css]: if possible use relative units for responsiveness

File: components/css/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    object-fit: cover;
+}
+.avatar-sm{
+    width:32px;
+    height:32px;
```
Comment (znjs) [inline components/css/avatar/avatar.css]: if possible use relative units for responsiveness

File: components/css/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+}
+.avatar-md{
+    width:64px;
+    height:64px;
+}
+.avatar-lg{
+    width:96px;
+    height:96px;
```
Comment (znjs) [inline components/css/avatar/avatar.css]: if possible use relative units for responsiveness

**Review by znjs — APPROVED** [review]

**Review by Nishant-96 — APPROVED** [review]

File: components/css/avatar/avatar.css (hunk 1 above)
Comment (rahulmistry751) [reply components/css/avatar/avatar.css]: Done. Thanks

File: components/css/avatar/avatar.css (hunk 3 above)
Comment (rahulmistry751) [reply components/css/avatar/avatar.css]: Done. Thanks

File: components/css/avatar/avatar.css (hunk 2 above)
Comment (rahulmistry751) [reply components/css/avatar/avatar.css]: Done. Thanks

**Pull request merged**